server/utils/errorHandler/errorWrapper.ts

```diff
@@ -0,0 +1,20 @@
+import { Request, Response, NextFunction, RequestHandler} from "express";
+
+const errorMappings: { [name: string]: number } ={
+  JsonWebTokenError: 401,
+  Unauthorized: 401,
+  ValidationError: 422,
+  "NotFound": 404
+};
+
+const errorWrapper = async(controller: RequestHandler,req:Request, res:Response, next:NextFunction) =>{
+  try {
+    await controller(req,res,next)
+  }
+  catch (error:any) {
```

Comment:
Using the unknown type in TypeScript is generally a better alternative than using the any type when you need to represent values of an unknown type.